Jenkinsfile

```diff
@@ -8,7 +8,7 @@ runTheBuilds.runDevToolsProject(
   test: {
     parallel(failFast: false,
       groovylint: {
-        groovylint.check('./Jenkinsfile,./src/**.groovy,*.gradle')
+        groovylint.check('./Jenkinsfile,**/*.gradle,**/*.groovy')
```

Comment:
I think that `./*.gradle` would be preferred. The other regexp is fine, though.

Reply:
If `./*.gradle` is preferred, then I guess we should change https://github.com/AbletonDevTools/postgres-pipeline-utils/blob/master/Jenkinsfile#L14 and https://github.com/AbletonDevTools/python-pipeline-utils/blob/master/Jenkinsfile#L15.

Reply:
I would prefer that, yes. However, let's double-check and confirm that it works first.